src/c/src/file.c

```diff
@@ -22,12 +22,34 @@
 
 #include "io.h"
 
+#ifdef _WIN32
+#define PN_FOPEN_MODE(s) L ## s
+#else
+#define PN_FOPEN_MODE(s) s
+#endif
+
+
+#ifdef _WIN32
+static FILE* pn_fopen_utf8(const char* path, const wchar_t* mode) {
+    pn_string16_t* wpath = pn_string16_from_utf8(path, strlen(path));
```

Comment:
There doesn’t seem to be an include for `common.h` from this file. WINE is failing but this seems to be just a warning in the native Windows build:
```
../../../src/c/src/file.c(34): warning C4047: 'initializing': 'pn_string16_t *' differs in levels of indirection from 'int'
```